cirq-google/cirq_google/cloud/quantum_v1alpha1/types/quantum.py

```diff
@@ -156,15 +156,48 @@ class QuantumJob(proto.Message):
 
 class DeviceConfigSelector(proto.Message):
     r"""-
+    This message has `oneof`_ fields (mutually exclusive fields).
```

Comment:
this is auto generated code?

Reply:
Yup, the changes in the file are from gapic client generation